build_defs/python.build_defs

```diff
@@ -47,6 +47,10 @@ def python_library(name:str, srcs:list=[], resources:list=[], deps:list=[], visi
         labels += ['py:zip-unsafe']
     if srcs or resources:
         cmd = '$TOOLS_JARCAT z -d -o ${OUTS} -i .'
+
+        # For some reason jarcat isn't unflattening .whl files so do it manually here till I figure out why
+        unzip_cmd = f'for i in $(find . -name \'*.whl\'); do mv $i .; unzip $(basename "$i"); done'
```

Comment:
would be nice to figure this out, this will be a significant performance disadvantage if we have to recompress a bunch of files.

Reply:
Ah I was missing a `-s .whl` in the jarcat command